site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -113,11 +119,44 @@ def _transform_titles(self):
 
         return rdm_title, rdm_additional_titles
 
+    def _validate_imprint(self):
+        """Validate that record has only 1 imprint."""
+        imprints = self.inspire_metadata.get("imprints", [])
+        if imprints:
+            if len(imprints) > 1:
+                self.metadata_errors.append(
+                    f"More than 1 imprint found. INSPIRE record id: {self.inspire_metadata.get('control_number')}."
+                )
+                return False
+            return imprints[0]
+        return False
```

Comment:
let's try to optimise for readability, how about avoiding nested conditional statements:
```suggestion
        if not imprints:
            return False
        if  len(imprints) > 1:
                self.metadata_errors.append(
                    f"More than 1 imprint found. INSPIRE record id: {self.inspire_metadata.get('control_number')}."
                )
                return False
        
        return imprints[0]
 
```